src/Composer/ScriptHandler.php

```diff
@@ -19,7 +19,7 @@ class ScriptHandler extends SensioScriptHandler
     /**
      * @var string
      */
-    const CORE_PACKAGE_NAME = 'phplist/phplist4-core';
+    const CORE_PACKAGE_NAME = 'phplist/core';
```

Comment:
This constant should move to the module finder, and the module finder should use it as well.